src/studio/src/designer/frontend/packages/schema-editor/src/components/Editor.test.tsx

```diff
@@ -166,13 +167,13 @@ test('should not show add property or add reference buttons on a reference node'
   const definitions = {
     mockDefinition: { type: 'object' },
   };
-  await renderEditor({
+  const { store, user } = renderEditor({
```

Comment:
`store` is unused and can be removed
```suggestion
  const { user } = renderEditor({
```